### ISSUE TITLE
feature request: schedule retweets

### ISSUE THREAD

**Issue by ingvald** [julien-c/Circular#31]
it would be great to be able to schedule retweets (not old-style RT quote). 

there's two ways i do that today via Buffer, and that is via a chrome extension adding an option on twitter.com, and via sending an email f.ex from my mobile twitter client.

**Comment by kxra** [comment]
yes, this is my primary use of buffer, only i use the firefox extension
